src/Form/ResidentComplaintBoardCaseType.php

```diff
@@ -41,11 +41,10 @@ public function buildForm(FormBuilderInterface $builder, array $options)
             ->add('complainantPostalCode')
             ->add('caseType', ChoiceType::class, [
                 'choices' => [
-                    $caseTypesAssociative,
+                    'Complaint Categories' => $caseTypesAssociative,
```

Comment:
Should come directly from the case entity object.

Reply:
That would not give the user any choice. But it needs modification after complaintCategory property addition on CaseEntity. 

Switched to EntityType and got 'choices' from Board.